src/test/rules/helper.ts

```diff
@@ -24,13 +25,16 @@ export function testScript(rule: string, scriptText: string, config: Object): bo
 /**
  * @deprecated Use ruleTester
  */
-export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: { rules: {} }) {
+export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: Lint.Configuration.IConfigurationFile) {
   if (!config) {
     config = {
-      rules: {}
+      rulesDirectory: [],
+      rules: new Map<string, Partial<IOptions>>(),
+      extends: [],
+      jsRules: new Map<string, Partial<IOptions>>()
     };
 
-    config.rules[rule] = true;
+    config.rules.set(rule, true);
```

Comment:
does that type check?
You should pass in an object with at least `ruleName` set